myhpi/search/templates/search/search_result.html

```diff
@@ -9,7 +9,7 @@ <h5 class="text-break">{{ result.title }}</h5>
                 {{ result.specific.date|date:"SHORT_DATE_FORMAT" }}
             </time>
         </hgroup>
-        <span class="text-dark">
+        <span class="search-preview">
             {% if result.search_description %}
                 {{ result.search_description }}
             {% else %}
```

Comment:
The selection in the results page now looks a bit off when pressing an item, due to now making the selection darker but keeping the same border and making the background white again

This PR:

![grafik](https://github.com/user-attachments/assets/1ddbdec2-22d7-4a13-81e0-83d7049f9937)

Production:

![grafik](https://github.com/user-attachments/assets/4ce9c6e6-e139-4f2a-a999-4eab1f339c81)